plugins/generator-1.15.2/forge-1.15.2/templates/gui.java.ftl

```diff
@@ -584,6 +584,29 @@ import ${package}.${JavaModName};
 		<#assign btid = 0>
         <#list data.components as component>
 			<#if component.getClass().getSimpleName() == "Button">
+			<#if component.externalLink?length == 0>
+			<#else>
+			Minecraft.getInstance().displayGuiScreen(new ConfirmOpenLinkScreen((open) -> {

```

Comment:
A better way to do that is to check if `component.externalLink?length` is not equal to 0

Reply:
I tried, but I didn't the thing. The documentation only says it returns the number of characters in the string.

Reply:
I found the solution on the documentation. It was only on another page.